docs/hello_nextflow/03_hello_workflow.md

```diff
@@ -793,7 +793,7 @@ But as the saying goes, why not both?
 
 ### 4.2. Report the output at the end of the workflow
 
-Now that we have two outputs coming out of the `collectGreetings` process, the `collectGreetings.out` output channel contains two 'tracks':
+Now that we have two outputs coming out of the `collectGreetings` process, the `collectGreetings.out` output multi-channel contains two channels:
```

Comment:
Oh yes this is a thing i misunderstood originally, good catch. But I would delete 'multi-channel' because it just complicates the sentence. "the output contains two channels" is clearer.